packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -1046,15 +1048,19 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
     }
   };
 
+  private _onDropdownMouseDown = (_ev: React.MouseEvent<HTMLDivElement>): void => {
```

Comment:
nit:

```suggestion
  private _onDropdownMouseDown = (ev: React.MouseEvent<HTMLDivElement>): void => {
```

Reply:
simply removed unused param

Reply:
For context, @xugao and I talked about this offline and decided that, because this is a private method, we have the liberty of removing the parameter now and adding it later if needed without affecting public facing API.